packages/controller-utils/src/index.test.ts

```diff
@@ -0,0 +1,75 @@
+import * as allExports from '.';
+
+describe('@metamask/controller-utils', () => {
+  it('has expected JavaScript exports', () => {
+    expect(Object.keys(allExports)).toMatchInlineSnapshot(`
+      Array [
+        "createServicePolicy",
```

Comment:
This should have been here all along, but this is where we are adding the new export.